### PR TITLE
Add relocker script for voter_msig

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackle the vlaura which gets over time unlock in the voter_msig, part of the convo at #626 

run:

```
brownie run badger/voter_processing_unlocks

brownie run badger/voter_chores
```

**Note:** opted for leaving in also the script which simply process the expired locks in case that fits on the timing, and we do need necessarily to trigger the big voter chore. imo is handy

Update on voter chores script steams from the convo at this [thread](https://discord.com/channels/1011322803414388897/1032235652307230771)

### CODE REVIEW

File: scripts/badger/voter_chores.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    GRAVI = voter.contract(r.sett_vaults.graviAURA)
+    AURA = voter.contract(r.treasury_tokens.AURA)
+
+    data = voter.badger.get_hh_data()
+    rewards = [x["token"] for x in data]
+
+    voter.take_snapshot(tokens=rewards)
+    destination.take_snapshot(tokens=rewards)
+
+    voter.badger.claim_bribes_hidden_hands(claim_from_strat=False)
```
Comment (gosuto-inzasheru) [inline scripts/badger/voter_chores.py:21]: this reverts on `VirtualMachineError: revert: Invalid _claims`. maybe add a check here as well?

File: scripts/badger/voter_chores.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    # add relock chore
+    _, unlockable, _, _ = vlAURA.lockedBalances(voter)
+    if unlockable > 0:
+        vlAURA.processExpiredLocks(True)
+
+        _, unlockable, _, _ = vlAURA.lockedBalances(voter)
+        assert unlockable == 0
+
+    # claim rewards and send to trops
+    vlAURA.getReward(voter)
```
Comment (gosuto-inzasheru) [inline scripts/badger/voter_chores.py:40]: this isnt being tracked by the safe snapshot

Comment (petrovska-petro) [reply scripts/badger/voter_chores.py:47]: added in the snap in 5d1f7e4e50ba4dbb2ef9b88b8e7e945d690f88a1

File: scripts/badger/voter_chores.py (hunk 1 above)
Comment (petrovska-petro) [reply scripts/badger/voter_chores.py]: include the check to see if there is indeed some data to claim for in 5d1f7e4e50ba4dbb2ef9b88b8e7e945d690f88a1

File: scripts/badger/voter_chores.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    AURABAL.transfer(destination, AURABAL.balanceOf(voter))
+
+    # wd from gravi and lock aura. gravi is coming as fee os processing bribes
+    gravi_balance = GRAVI.balanceOf(voter)
+    if gravi_balance > 0:
+        GRAVI.withdrawAll()
```
Comment (gosuto-inzasheru) [inline scripts/badger/voter_chores.py:53]: there is not enough liq in the vault to withdraw the current amount. kinda nasty, but i think we should do like a 75% withdrawal of the vault's balance in that case instead.

too bad there isnt a withdrawAsMuchAsPossible()...

File: scripts/badger/voter_processing_unlocks.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
```
Comment (gosuto-inzasheru) [inline scripts/badger/voter_processing_unlocks.py:5]: lets incorp this into `voter_chores` as well as to not have redundant code

Comment (petrovska-petro) [reply scripts/badger/voter_processing_unlocks.py]: resolved in bef27e9818c2f9d02ff5892f422df72e78f50250

File: scripts/badger/voter_chores.py (hunk 3 above)
Comment (petrovska-petro) [reply scripts/badger/voter_chores.py]: push a safer approach in 7914726d9854af03e8c65ec75afa220564a0c462

**Review by gosuto-inzasheru — APPROVED** [review]
runs good on my local.

`WD_PROTECTION` might be a bit tight, but we can always adjust to `.9` or whatever when running in production

**Review by jalbrekt85 — APPROVED** [review]
covers:
- claim hh bribes
- claim vlaura rewards
- relock vlaura
- withdraw graviaura
- lock aura
- sweep remaining erc20s to trops

lgtm

**Pull request merged**